readme.md

```diff
@@ -1,5 +1,7 @@
 # todo-list
-This is a simple to-do list application built with JavaScript, HTML, and CSS. It allows users to create a list of tasks with a description, completion status, and index.
+
+This simple to-do list application is built with JavaScript, HTML, and CSS. It allows users to create a list of tasks with a description, completion status, and index.
+
 
 # 📗 Table of Contents
 
```

Comment:
- **Hi, There is a slight mistake in your readme template. I can see two tables of content. Can you fix this? This will avoid confusion.** 
- 
![Capture](https://github.com/misikir21/my-todo-list/assets/63972670/25898562-15bb-4dc4-9a6b-2a3526345dba)